src/mixins/filing-template-mixin.ts

```diff
@@ -163,20 +163,16 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
             : {}
         },
         courtApproval: this.getAmalgamationCourtApproval,
-        parties: fixNullAddressType(this.getAddPeopleAndRoleStep.orgPeople)
+        parties: fixNullAddressType(this.getAddPeopleAndRoleStep.orgPeople),
+        shareStructure: {
+          shareClasses: this.getCreateShareStructureStep.shareClasses
+        },
+        incorporationAgreement: {
```

Comment:
Do you think we can remove this `incorprationAgreement` property? I didn't add it to Continuation In as well as I think it's IAs only.

Reply:
Hey, yeah! We don't prompt for this data so there's no point in saving (or restoring) it.

Reply:
Fixed, thanks! Resolving.